### ISSUE TITLE
kodi 18

### ISSUE THREAD

**Issue by vTneilA** [braz96/skin.mimic#74]
hi there... is this skin working on kodi 18 ?

**Comment by braz96** [comment]
Yes, the master branch is for Kodi 18. It's also in the Kodi 18 repo.

**Comment by vTneilA** [comment]
im not sure but there is a issue when u will open configuration any addons... just check it out plz.. go to system - addons - my addons - tyr open any addon and go to configuration of cours on minic skin..

**Comment by braz96** [comment]
Known issue, it is fixed on GitHub.

**Comment by Vidalcris** [comment]
I got the addon setting bug for some days now..
At first that was working fine but now even if i reset the skin i cannot access any addon settings.
When i go back to another skin, this is working again.
I tried to install latest kodi 18 without any changes.

**Comment by untoreh** [comment]
had to reset to default/delete addon data
